src/Servers/Kestrel/Core/src/Http2Limits.cs

```diff
@@ -141,5 +141,15 @@ public int InitialStreamWindowSize
                 _initialStreamWindowSize = value;
             }
         }
+
+        /// <summary>
+        /// Gets or sets a value that controls whether dynamic compression of response headers is allowed.
+        /// For more information about the security considerations of HPack dynamic header compression, visit
+        /// https://tools.ietf.org/html/rfc7541#section-7.
+        /// </summary>
+        /// <remarks>
+        /// Defaults to true.
+        /// </remarks>
+        public bool AllowResponseHeaderCompression { get; set; } = true;
```

Comment:
This seems more like general config that a "limit". I think this should move to KestrelServerOptions. It probably makes sense to have this setting apply also to QPACK if and when we add dynamic compression support for that.

Reply:
😆 I gave exactly the opposite advice earlier. The theory was that A) QPACK hasn't been implemented and isn't likely to be any time soon, and B) if you're turning it off due to a security concern that is likely to be HTTP/2 specific.

Reply:
It's not really even a limit in my mind. And if you want it off in HTTP/2, I don't know why you wouldn't in HTTP/3.

This would be the first bool property in either Http2Limits or Http3Limits. I feel this belongs more with options like DisableStringReuse and EnableAltSvc in  KestrelServerOptions.